readthedocs/storage/s3_storage.py

```diff
@@ -110,20 +108,7 @@ class NoManifestS3StaticStorage(S3StaticStorageMixin, OverrideHostnameMixin, S3B
     internal_redirect_root_path = "proxito-static"
 
 
-class S3BuildEnvironmentStorage(S3PrivateBucketMixin, S3BuildMediaStorageMixin):
-    bucket_name = getattr(settings, "S3_BUILD_ENVIRONMENT_STORAGE_BUCKET", None)
-
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-
-        if not self.bucket_name:
-            raise ImproperlyConfigured(
-                "AWS S3 not configured correctly. "
-                "Ensure S3_BUILD_ENVIRONMENT_STORAGE_BUCKET is defined.",
-            )
-
-
-class S3BuildToolsStorage(S3PrivateBucketMixin, S3BuildMediaStorageMixin):
+class S3BuildToolsStorage(S3PrivateBucketMixin, S3Boto3Storage):
```

Comment:
This class doesn't need to inherit from `S3BuildMediaStorageMixin`, so I just merged that mixin with the only class that actually needs to set the _rclone attribute.